.travis.yml

```diff
@@ -13,33 +13,36 @@ script:
   - 'bundle exec rake $CHECK'
 bundler_args: --without system_tests
 rvm:
-  - 2.4.1
+  - 2.5.0
```

Comment:
We're currently shipping ruby 2.5.1, see [component versions](https://puppet.com/docs/pe/2019.0/component_versions_in_recent_pe_releases.html), please update the settings used here.